syft/frameworks/torch/functions.py

```diff
@@ -1,7 +1,9 @@
 from .tensors.interpreters import MultiPointerTensor
+from syft.frameworks.torch.pointers.object_pointer import ObjectPointer
+from typing import List
 
 
-def combine_pointers(*pointers):
+def combine_pointers(*pointers: List[ObjectPointer]) -> MultiPointerTensor:
```

Comment:
you're missing a close paren

Reply:
I don't see it, where?

Reply:
Why isn't this being merged? What am I missing?

Reply:
The close paren is there, there's just a return type annotation that hides it a bit. Suggest resolving this thread

Reply:
Well,that is how the formatter (black) formatted it. Should I change it then ? @jvmancuso

Reply:
No, I'm saying there's no change needed here, so @robert-wagner can clear this requested change when he's able

Reply:
@jidroid404 apparently this file still needs to be merged.

Reply:
My apologies @jjdroid404 . I read the line in correctly